data/json/items/melee/swords_and_blades.json

```
@@ -287,7 +287,7 @@
     "id": "bone_knife",
     "name": { "str": "bone shiv" },
     "type": "TOOL",
-    "category": "tools",
+    "category": "tools_farming",
```

This one's especially goofy, it's the least knife-y of all the knives. XD
```suggestion
    "category": "tools_workshop",
```